### PR TITLE
Added "How to Choose" section and "Choose if..." row

### PR DESCRIPTION
**Pull request by HPrinz**

### CODE REVIEW

File: _includes/comparison.html [hunk 1]
```diff
@@ -50,24 +50,58 @@ <h2 class="section-heading--primary"> Technical Comparison </h2>
...
+          <td><a href="https://linkerd.io/2/tasks/books/" target="_blank">Linkerd Tasks</a></td>
+          <td><a href="docs.aws.amazon.com/app-mesh/latest/userguide/appmesh-getting-started.html" target="_blank">AWS App Mesh Getting Started</a></td>
+          <td><a href="https://learn.hashicorp.com/consul/getting-started/connect" target="_blank">Hashicorp Learn platform</a></td>
+          <td><a href="https://docs.mae.sh/examples/" target="_blank">Maesh Example</a></td>
+          <td><a href="https://kuma.io/docs/0.2.2/installation/kubernetes/#kubernetes" target="_blank">Kuma Kubernetes Quickstart</a></td>
+        </tr>
     </tbody>
+    <thead style="background-color: white; color: #ff4d67">
+        <tr>
+          <th scope="col">Compatibility, Performance & Conplexity</th>
```
Comment (vanto) [inline _includes/comparison.html]: s/Conplexity/Complexity
```suggestion
          <th scope="col">Compatibility, Performance & Complexity</th>
```

Comment (HPrinz) [reply _includes/comparison.html]: Thank you! 👍

File: _includes/comparison.html [hunk 2]
```diff
@@ -360,6 +354,31 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
...
     </tbody>
+    <thead style="background-color: white; color: #ff4d67">
+      <tr>
+        <th scope="col">Advantages & Drawbacks</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th scope="row">Advantages</th>
+        <td>Istio can be adapted and extended like no other Mesh. Thanks to it's big and vivid community, it's constantly improved. Istio is <a href="https://istio.io/about/community/customers/" target="_blank">used in production.</a></td>
```
Comment (ewolff) [inline _includes/comparison.html]: Do you feel that its support for the different platforms is worth mentioning?

File: _includes/comparison.html [hunk 3]
```diff
@@ -360,6 +354,31 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
...
+        <td>Linkerd 2 is designed to be non-invasive and is optimized for performance and usability. It therefore requires little time to adopt. Linkerd 2 is <a href="https://github.com/linkerd/linkerd2/blob/master/ADOPTERS.md" target="_blank">used in production.</a></td>
+        <td>AWS App Mesh is integrated in the AWS landscape and it is fully managed for you.</td>
+        <td>Consul Connect is well integrated with Consul and the Proxy can be changed and extended.</td>
```
Comment (ewolff) [inline _includes/comparison.html]: As it uses Consul it should be possible to run it on any Docker scheduler, right? Is that worth mentioning?

File: _includes/alternatives.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+    <div class="list-teaser list-teaser__body">
+      <h2 class="list-teaser__headline">No (synchronous) Microservices</h2>
+      <p class="list-teaser__text">
+        Synchronously integrated microservices are a popular architecture, despite several drawbacks such as latency and operational complexity. 
+        One of the reasons is that no other architecture facilitate faster delivery of software.
+        <br>
+        <br>
+        What is often overseen is that there are more approaches to microservices than integrating them synchronously. 
+        Patterns like SCS and asynchronous communication aim to mitigate many problems of classic mircoservices.
```
[the remark this answers is not in the document]
Comment (HPrinz) [reply _includes/alternatives.html]: Theoretically there is, but (understandably) the Service Meshes focus on support for sync microservices. 
This section should express that a Service Mesh might be a solution for a problem that is rooted somewhere else and should also be solved elsewhere.

File: _includes/comparison.html (hunk 2 above)
Comment (HPrinz) [reply _includes/comparison.html]: Absolutely, thanks

File: _includes/comparison.html (hunk 3 above)
Comment (HPrinz) [reply _includes/comparison.html]: Yep, good point. I'll add a sentence there too

File: _includes/alternatives.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (ewolff) [reply _includes/alternatives.html]: I see. I thought you meant to say that service meshes are useless for async microservices. Maybe you want to add a paragraph that explains how service meshes might help for async?

[the remark this answers is not in the document]
Comment (ewolff) [reply _includes/alternatives.html]: I see. I thought you meant to say that service meshes are useless for async microservices. Maybe you want to add a paragraph that explains how service meshes might help for async?

[the remark this answers is not in the document]
Comment (ewolff) [reply _includes/alternatives.html]: I see. I thought you meant to say that service meshes are useless for async microservices. Maybe you want to add a paragraph that explains how service meshes might help for async?

[the remark this answers is not in the document]
Comment (HPrinz) [reply _includes/alternatives.html]: Yes, I was about to propose that :) 
I'll add an issue

**Pull request merged**